submit-web/src/components/Shared/Popups/ConfirmationDialog.tsx

```diff
@@ -1,25 +1,39 @@
-import { Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle, Button } from '@mui/material';
+import {
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Button,
+} from "@mui/material";
+import { useModal } from "../Modals/modalStore";
 
 type ConfirmationDialogProps = {
-  isOpen: boolean;
   title: string;
   description: string;
   onConfirm: () => void;
-  onCancel: () => void;
 };
 
-const ConfirmationDialog: React.FC<ConfirmationDialogProps> = ({ isOpen, title, description, onConfirm, onCancel }) => {
+const ConfirmationDialog: React.FC<ConfirmationDialogProps> = ({
```

Comment:
shouldn't we be using this component here https://mui.com/material-ui/react-snackbar/

Reply:
we can